modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/BlogPostingResourceImpl.java

```diff
@@ -341,10 +341,15 @@ private BlogPosting _addBlogPosting(
 	private ServiceContext _createServiceContext(
 		BlogPosting blogPosting, long groupId) {
 
-		return ServiceContextRequestUtil.createServiceContext(
-			blogPosting.getTaxonomyCategoryIds(), blogPosting.getKeywords(),
-			_getExpandoBridgeAttributes(blogPosting), groupId,
-			contextHttpServletRequest, blogPosting.getViewableByAsString());
+		ServiceContext serviceContext =
+			ServiceContextRequestUtil.createServiceContext(
+				blogPosting.getTaxonomyCategoryIds(), blogPosting.getKeywords(),
+				_getExpandoBridgeAttributes(blogPosting), groupId,
+				contextHttpServletRequest, blogPosting.getViewableByAsString());
+
+		serviceContext.setAssetPriority(blogPosting.getPriority());
```

Comment:
I assume that we are putting the priority on the serviceContext because it will be used by the service, right?

Reply:
Exactly; the BlogsEntry service is using the context for managing the asset related fields

Reply:
> Exactly; the BlogsEntry service is using the context for managing the asset related fields

Exactly; the BlogsEntry service is using the context for managing the asset related fields